src/zero/api/handlers.clj

```diff
@@ -51,6 +51,15 @@
         results (wgs/submit-some-workflows env max input_path output_path)]
     (succeed {:results results})))
 
+(defn append-to-aou-workload
```

Comment:
The ability to add samples to an existing workload might be useful to have for other pipelines as well! But this can be a future discussion with Ops.

Reply:
yes! I think we could make make this an implementation of the `update-workload!` multi-method and apply it to all types of workloads (pipelines)! I agree this might be better in another PR after discussing with hOps